lib/backup_restore/backuper.rb

```diff
@@ -321,9 +321,19 @@ def notify_user
       log "Notifying '#{@user.username}' of the end of the backup..."
       status = @success ? :backup_succeeded : :backup_failed
 
-      post = SystemMessage.create_from_system_user(
-        @user, status, logs: Discourse::Utils.pretty_logs(@logs)
-      )
+      upload = Logger.save_log_to_upload(user: @user, logs: @logs)
+      if upload.persisted?
+        logs = UploadMarkdown.new(upload).attachment_markdown
+      else
+        Rails.logger.warn("Failed to upload the backup logs file: #{upload.errors.full_messages}")
+        logs = <<~RAW
+          ```text
+          #{upload.errors.full_messages}
```

Comment:
I feel like the error message here is not going to be useful for the user. Perhaps we should just trancate the logs to a sane length should there be an upload failure?

Reply:
I updated the PR to include the backup logs whenever it is possible.